venues/ICLR.cc/2018/Conference/webfield/conferenceWebfield_alt.js

```diff
@@ -303,6 +307,45 @@ function renderContent(allNotes, submittedNotes, assignedNotePairs, userGroups,
     Webfield.setupAutoLoading(BLIND_INVITATION, PAGE_SIZE, submissionListOptions);
   }
 
+  // Withdrawn Papers tab
+  var withdrawnListOptions = _.assign({}, paperDisplayOptions, {
+    showTags: false,
+    container: '#withdrawn-papers'
+  });
+
+  if (withdrawnNotes.length) {
+    Webfield.ui.submissionList(withdrawnNotes, {
+      heading: null,
+      container: '#withdrawn-papers',
+      search: {
+        enabled: true,
+        subjectAreas: SUBJECT_AREAS_LIST,
+        onResults: function(searchResults) {
+          var withdrawnSearchResults = searchResults.filter(function(note) {
+            return note.invitation === WITHDRAWN_INVITATION;
+          });
+          Webfield.ui.searchResults(withdrawnSearchResults, withdrawnListOptions);
+          Webfield.disableAutoLoading();
+        },
+        onReset: function() {
+          Webfield.ui.searchResults(withdrawnNotes, withdrawnListOptions);
+          if (withdrawnNotes.length === PAGE_SIZE) {
+            Webfield.setupAutoLoading(WITHDRAWN_INVITATION, PAGE_SIZE, withdrawnListOptions);
+          }
+        }
+      },
+      displayOptions: withdrawnListOptions,
+      fadeIn: false
+    });
+  } else {
+    $('.tabs-container a[href="#withdrawn-papers"]').parent().hide();
+  }
+
+
+  if (withdrawnNotes.length === PAGE_SIZE) {
+    Webfield.setupAutoLoading(WITHDRAWN_INVITATION, PAGE_SIZE, withdrawnListOptions);
```

Comment:
I don't think you need to set up infinite scroll either, it's not like people are going to be withdrawing more than 50 papers.